blivet/devicelibs/dasd.py

```diff
@@ -114,6 +114,10 @@ class dasd_info(ctypes.Structure):
 
 def is_ldl_dasd(device):
     """Determine whether or not a DASD is LDL formatted."""
+    statusfile = "/sys/block/%s/device/status" % (device,)
+    if not os.path.isfile(statusfile):
+        return False
```

Comment:
As far as s390x devices, yes. Considering is_ldl_dasd isn't going to run on non-s390x arches, I thought this was reasonable, as it's used elsewhere (make_unformatted_dasd_list I think).